api-gateway/api-gw/routes.yaml

```diff
@@ -2,10 +2,10 @@
 apiVersion: gateway.networking.k8s.io/v1alpha2
 kind: HTTPRoute
 metadata:
-  name: test-route-1
+  name: example-route-1
 spec:
   parentRefs:
-  - name: test-gateway
+  - name: example-gateway
```

Comment:
heads up that I changed the `test-*` names to be `example-*` since it's not really "testing" at that point